tests/performance/mmap_vs_directio/mmap_vs_directio.rb

```diff
@@ -157,7 +165,7 @@ def benchmark_queries(query_file, type, clients, warmup = false, runtime = 20)
     run_fbench2(@container,
                 node_file,
                 {:clients => clients,
-                 :append_str => '&presentation.summary=minimal&hits=10',
+                 :append_str => "&presentation.summary=#{summary}&hits=10",
```

Comment:
`presentation.summary` should always be `minimal` for this test. However, we need an extra parameter: `&ranking.profile` which is `default` by default and `use_bitvectors` in the filter test.